tech-summit-demo/launch_demo.py

```diff
@@ -0,0 +1,320 @@
+"""Inegration tests script."""
```

Comment:
integration

Reply:
renamed this comment to DLT META DAIS DEMO